src/pages/remotes.ts

```diff
@@ -7,3 +7,11 @@ export function getMe() {
 export interface GetMe {
   name: string;
 }
+
+export function getCards() {
+  return http.get<GetCards>('/api/cards');
+}
+
+export interface GetCards {

```

Comment:
interface는 데이터의 생김새를 정의하는 것이라 행위가 포함되는 것이 어색하게 느껴집니다.
```suggestion
export interface CardsResponse {

```
와 같이 인터페이스가 나타내고자 하는 데이터 자체에 집중한 이름은 어떠신가요?